cloud/azure/main.tf

```diff
@@ -206,6 +119,55 @@ resource "azurerm_log_analytics_workspace" "civiform_logs" {
   retention_in_days   = 30
 }
 
+output "app_service_default_hostname" {
+  value = "https://${azurerm_app_service.civiform-app.default_site_hostname}"
+}
+
+resource "azurerm_monitor_diagnostic_setting" "app_service_log_analytics" {
+  name                       = "app_service_log_analytics"
+  target_resource_id         = azurerm_app_service.civiform-app.id
+  log_analytics_workspace_id = azurerm_log_analytics_workspace.civiform_logs.id
+
+  log {
+    category = "AppServiceAppLogs"
+
+    retention_policy {
+      enabled = false
+    }
+  }
+
+  log {
+    category = "AppServiceConsoleLogs"
+
+    retention_policy {
+      enabled = false
+    }
+  }
+
+  log {
+    category = "AppServiceHTTPLogs"
+
+    retention_policy {
+      enabled = false
+    }
+  }
+
+  log {
+    category = "AppServiceAuditLogs"
+
+    retention_policy {
+      enabled = false
+    }
+  }
+  metric {
+    category = "AllMetrics"
+
+    retention_policy {
+      enabled = false
```

Comment:
question (blocking): does setting `enabled = false` on all of these mean we won't be able to see them?

Reply:
No, it means there's no specific retention policy for the data type and they'll inherit the policy for the workspace. (See: https://docs.microsoft.com/en-us/azure/azure-monitor/logs/manage-cost-storage). We probably should eventually set a reasonable retention policy so the storage costs don't balloon up. Probably something like 60-90 days?

Reply:
Ah got it, thanks! I'd say 30 days is alright for now and we can make it configurable. The lower the less expensive.